src/main/scala/amba/axi4/Deinterleaver.scala

```diff
@@ -81,13 +85,15 @@ class AXI4Deinterleaver(maxReadBytes: Int)(implicit p: Parameters) extends LazyM
 
         // Transmit the selected burst to inner
         in.r.valid := locked
-        in.r.bits  := Vec(qs.map(_.deq.bits))(deq_id)
+        val deq_bits = WireInit(VecInit(qs.map(_.deq.bits)))
+        in.r.bits  := deq_bits(deq_id)
```

Comment:
Was changing Vec to VecInit not enough?

Reply:
TBH i didn't try it. I will try it out. I was going by memory of fixing these things before, but maybe that's only needed for RegInit

Reply:
`VecInit` should be sufficient

Reply:
confirmed